src/analyzer.js

```diff
@@ -10,6 +10,7 @@ const {parseBundle} = require('./parseUtils');
 const {createAssetsFilter} = require('./utils');
 
 const FILENAME_QUERY_REGEXP = /\?.*$/;
+const FILENAME_EXTENSIONS = /.(js|mjs)$/i;
```

Comment:
This would match `fooXjs`, too: You will need to escape the `.` as it matches any character:

```suggestion
const FILENAME_EXTENSIONS = /\.(js|mjs)$/i;
```